integration_tests/src/main/python/parquet_test.py

```diff
@@ -62,7 +62,8 @@ def test_parquet_fallback(spark_tmp_path, read_func, disable_conf):
     assert_gpu_fallback_collect(
             lambda spark : reader(spark).select(f.col('*'), f.col('_c2') + f.col('_c3')),
             'FileSourceScanExec',
-            conf={disable_conf: 'false'})
+            conf={disable_conf: 'false',
+                "spark.sql.sources.useV1SourceList": "parquet"})
```

Comment:
Do we need/want follow on issues for datasource V2 vs V1 testing?

Reply:
yeah I'll file one

Reply:
https://github.com/NVIDIA/spark-rapids/issues/829